docs/network-edge/gslb.mdx

```diff
@@ -0,0 +1,205 @@
+import Tabs from "@theme/Tabs";
+import TabItem from "@theme/TabItem";
+
+import ExplicitRegionAgentCliExample from "/examples/agent-cli/explicit-region.mdx";
+import ExplicitRegionAgentConfigExample from "/examples/agent-config/explicit-region.mdx";
+import ExplicitRegionSshExample from "/examples/ssh/explicit-region.mdx";
+import ExplicitRegionGoSdkExample from "/examples/go-sdk/explicit-region.mdx";
+import ExplicitRegionRustSdkExample from "/examples/rust-sdk/explicit-region.mdx";
+import ExplicitRegionKubernetesExample from "/examples/k8s/explicit-region.mdx";
+
+# Global Server Load Balancing
+
+---
+
+## Overview
+
+Global Server Load Balancing (GSLB) improves the performance and resiliencey of
```

Comment:
Global Server Load Balancing (GSLB) improves the performance and resiliencey of
your applications by distributing traffic to the geographically-nearest healthy
point of presence from the perspective of the connecting client.

Is it geographically nearest? Or should we say '...by distributing traffic to the point of presence with the lowest latency...